Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -304,7 +305,13 @@ private void AddModulePanes()
                 {
                     ClassName += " m2vDisplayNone";
                 }
-                ContentPane.Controls.Add(ParseControl("<div class=\"" + ClassName + "\"><div id=\"vjEditor\">" + InjectModules(html.DocumentNode.OuterHtml) + "</div></div>"));
+
+                try
+                {
+                    ContentPane.Controls.Add(ParseControl("<div class=\"" + ClassName + "\"><div id=\"vjEditor\">" + InjectModules(html.DocumentNode.OuterHtml) + "</div></div>"));
+                }
+                catch (Exception ex) { Exceptions.LogException(ex); }
```

Comment:
This doesn't add a generic error message on the page indicating there is an issue with the content. Maybe use the following message. 

Show the following message (localized) to non-admin users:
---------
Content in this section is currently unavailable. Please contact the site administrator if the issue persists. 

Show the following message (localized) to admin/host users:
---------
An error occurred in displaying content for this section. Please check event viewer for more information. You may also revert back to a previous version.

Reply:
The error message must also be formatted so it looks like an error message.

Reply:
![image](https://user-images.githubusercontent.com/62532215/102325382-150b0a00-3fa9-11eb-8171-6a3d9fcbd3bb.png)
![image](https://user-images.githubusercontent.com/62532215/102326274-36202a80-3faa-11eb-8821-72ff999cae14.png)